### DISCUSSION TITLE
Can i get the startruck badge in this situation ?

### DISCUSSION THREAD

**Discussion by mouadessalim** [community/community#29460] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey, hope all the github community are good. So i was wondering if i can have more clarification about my situtation. A few week ago i've decided to create with a mate a new github repo. So to do that we created a github organition with me and him. After creating the github repository ans the project, my mate started to lose interest in the project so I decided to transfer it to my personal account. Actually the project has 14 stars. My question is : can i get the startruck badge when the project reach 16 stars iven if it's a transfered repo (bideway i've writted 95% of the code).

**Comment by radar-blip** [comment]
Here is some info about the Starstruck achievement badge (and others) :
https://github.com/Schweinepriester/github-profile-achievements#achievements
You'll have to wait until you get 16 stars.

**Reply by mwihoti** [reply]
You get Starstruck achievement badge when you have 16 stars

**Comment by lbirkert** [comment, marked as answer]
When a Repository on your account hits 16 stars, even if it was transfered, you will receive the badge.

Hope this helps

**Reply by kylebrowning** [reply]
What about if its transferred the other way. I made a repo that has 500 stars, transferred it to a community, will I get startruck when it hits 516?

**Reply by webs3dev** [reply]
yea trying to get it g

**Reply by hotcoding85** [reply]
https://github.com/hotcoding85/ckeditor5

pls check and give me a star

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by mndree** [comment]
yes it can, you just have to wait to it reach 16 stars or you can ask your friend to give your repository a star

here is some list that maybe can help you 
https://github.com/drknzz/GitHub-Achievements

pls set my reply as your answer so i can get a achievement @mouadessalim hahahaha

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by hotcoding85** [comment]
https://github.com/hotcoding85/ckeditor5

Pls check and give me a star 😊

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by vunguyenkhanh** [comment]
https://github.com/vunguyenkhanh/vunguyenkhanh
Please check and give me a Star 😊

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by VArtzy** [comment]


**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by dripton-art** [comment]
Can I also earn a starstruck badge by giving Stars to other people's repos?
@github-staff

**Reply by khalifarsm** [reply]
https://github.com/khalifarsm/botox-android

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by khalifarsm** [comment]
https://github.com/khalifarsm/botox-android

**Comment by MasteraSnackin** [comment]
https://github.com/MasteraSnackin/github-badges-test

**Reply by YarinsHyper** [reply]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼

**Comment by YarinsHyper** [comment]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼
